### PR TITLE
ZKUI-100 - improve keys header for iam users

### PR DESCRIPTION
**Pull request by MonPote**

### CODE REVIEW

File: src/react/account/AccountUserAccessKeys.jsx [hunk 1]
```diff
@@ -143,6 +169,46 @@ const AccountUserAccessKeys = () => {
...
+          title="Warning"
+        >
+          Security Status: as a best practice, an user should not have more than
+          2 Access keys
+        </Banner>
+      </>
+    );
+  } else {
+    accessKeysCountComponent = `Access Keys ${accessKeysResult?.length ?? 0}`; // eslint-disable-line
+  }
```
Comment (JBWatenbergScality) [inline src/react/account/AccountUserAccessKeys.jsx]:
```suggestion
  const accessKeysResultLength = accessKeysResult?.length ?? 0;
  const accessKeysCountComponent = useMemo(() => {
    if (accessKeysStatus === 'loading' || accessKeysStatus === 'idle') {
    return 'Loading access keys...';
  } else if (accessKeysResultLength > 2) {
    return (
      <>
        Access Keys
        <TextBadge
          variant={'statusWarning'}
          text={accessKeysResultLength}
          style={{ margin: `0 ${spacing.sp24} 0 ${spacing.sp12}` }}
        />
        <Banner
          icon={
            <div
              style={{
                display: 'flex',
                alignItems: 'center',
                marginLeft: spacing.sp8,
              }}
            >
              <Icon
                name={'Exclamation-circle'}
                size={'lg'}
                color={'statusWarning'}
              />
            </div>
          }
          variant="statusWarning"
          title="Warning"
        >
          Security Status: as a best practice, an user should not have more than
          2 Access keys
        </Banner>
      </>
    );
  } else {
    return `Access Keys ${accessKeysResultLength}`;
  }, [accessKeysStatus, accessKeysResultLength]);
  
```

File: src/react/account/AccountUserAccessKeys.jsx [hunk 2]
```diff
@@ -143,6 +169,46 @@ const AccountUserAccessKeys = () => {
...
+          variant="statusWarning"
+          title="Warning"
+        >
+          Security Status: as a best practice, an user should not have more than
+          2 Access keys
+        </Banner>
+      </>
+    );
+  } else {
+    accessKeysCountComponent = `Access Keys ${accessKeysResult?.length ?? 0}`; // eslint-disable-line
```
Comment (JBWatenbergScality) [inline src/react/account/AccountUserAccessKeys.jsx]: We likely want the length to be in a badge, @Cuervino do you confirm?

**Review by JBWatenbergScality — APPROVED** [review]

**Comment by MonPote** [comment]
ping

**Pull request merged**